tensorflow/lite/micro/kernels/arc_mli/mli_tf_utils.h

```diff
@@ -93,15 +94,19 @@ static inline void ConvertToMliQuantParamsPerChannel(const TfLiteTensor* tfT,
 }
 
 template <typename datatype>
-static void ConvertToMliTensor(const TfLiteTensor* tfT, mli_tensor* mliT) {
-  ConvertToMliTensorData<datatype>(tfT, mliT);
+inline void MliTensorAttachBuffer(const TfLiteEvalTensor* tfT,
+                                  mli_tensor* mliT) {
+  mliT->data = (void*)tflite::micro::GetTensorData<datatype>(tfT);
```

Comment:
prefer C++-style cast. In this case you may not need to cast at all.

Reply:
I'd rather leave cast to have it explicit.